src/main/scala/com/microsoft/hyperspace/util/FileUtils.scala

```diff
@@ -68,8 +69,9 @@ object FileUtils {
    *
    * @param dirPath the directory path.
    */
-  def createDirectory(dirPath: Path): Unit = {
-    val fileSystem = dirPath.getFileSystem(new Configuration)
+  def createDirectory(dirPath: Path,
+      hadoopConfiguration: Configuration = new Configuration): Unit = {
+    val fileSystem = dirPath.getFileSystem(hadoopConfiguration)
```

Comment:
Are you using `scalafmt`? I get the following if I auto-format:
```scala
  def createDirectory(
      dirPath: Path,
      hadoopConfiguration: Configuration = new Configuration): Unit = {
    val fileSystem = dirPath.getFileSystem(hadoopConfiguration)
    if (!fileSystem.exists(dirPath)) {
      fileSystem.mkdirs(dirPath)
    }
  }
```

Reply:
Can you check other places as well?

Reply:
@imback82 I'm using the `scalastyle-config.xml` file present in the project. 

I tried setting **scalafmt** but it needs a `scalafmt.conf` config file. Is that present somewhere in the project?

Reply:
@andrei-ionescu Please use this: https://github.com/microsoft/hyperspace/blob/master/dev/.scalafmt.conf

Reply:
@imback82 Ultimately I did found the `.scalafmt.conf` file in `dev` folder. Will try to apply it.

Reply:
There seems to be some sort of misconfiguration in the scalafmt conf file because when I run it with optimise imports on a file it reorders the imports putting the `com.microsoft.hyperspace.*` at the top.